### DISCUSSION TITLE
[BUG] Web Admin: Al cambiar imagen de canal: selector aparece dos veces.

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#366] [4. Bug Reports] [answered]
Hola a todos

Dejo el video que explicaría más que el texto.
**Link:** https://drive.google.com/file/d/1t-h8ybDO60wU0xghye6CSoO0hVuzJydQ/view?usp=sharing

Gracias =>

**Comment by acedward** [comment, marked as answer]
Nota: no veo el error en el video.
Pero entiendo el problema. 
Marcado como BUG.

**Reply by NicolasPLG** [reply]
gracias @acedward, efectuvamente no se grabó la parte que selecciono la imagen y por eso no salió, pero que bueno que se entendió el problema.

Muchas gracias =>.

Cerraré el tema
